api/schema_system.go

```diff
@@ -189,3 +239,72 @@ func UnmarshalRequiredColumnRulePayload(payload string) (*RequiredColumnRulePayl
 	}
 	return &rcr, nil
 }
+
+// ProdTemplateSchemaReviewPolicy returns the default schema review policy.
```

Comment:
If these are used in tests only for the moment, let's keep them in the tests.go.